charts/federation-v2/values.yaml

```diff
@@ -34,6 +34,8 @@ controllermanager:
   ## Supported options are `configmaps` and `endpoints`
   leaderElectResourceLock:
   ## Value of feature gates item should be either `true` or `false`
+  syncController:
```

Comment:
This should be above the comment on 36 which is intended to document the feature gates values.

Reply:
Fixed.